src/main.c

```diff
@@ -202,232 +208,234 @@ typedef struct{
+#define SE_REGION_KEY_L_PRESSED 5
+#define SE_REGION_KEY_R 6
+#define SE_REGION_KEY_R_PRESSED 7
+#define SE_REGION_KEY_START 8
+#define SE_REGION_KEY_START_PRESSED 9
+#define SE_REGION_KEY_SELECT 10
+#define SE_REGION_KEY_SELECT_PRESSED 11
+#define SE_REGION_KEY_A 12
+#define SE_REGION_KEY_A_PRESSED 13
+#define SE_REGION_KEY_B 14
+#define SE_REGION_KEY_B_PRESSED 15
+#define SE_REGION_KEY_X 16
+#define SE_REGION_KEY_X_PRESSED 17
+#define SE_REGION_KEY_Y 18
+#define SE_REGION_KEY_Y_PRESSED 19
+#define SE_REGION_KEY_TURBO 20
+#define SE_REGION_KEY_TURBO_PRESSED 21
+#define SE_REGION_KEY_HOLD 22
+#define SE_REGION_KEY_HOLD_PRESSED 23
+#define SE_REGION_KEY_BLANK 24
+#define SE_REGION_KEY_BLANK_PRESSED 25
+#define SE_REGION_DPAD_UL 26
+#define SE_REGION_DPAD_UP 27
+#define SE_REGION_DPAD_UR 28
+#define SE_REGION_DPAD_LEFT 29
+#define SE_REGION_DPAD_CENTER 30
+#define SE_REGION_DPAD_RIGHT 31
+#define SE_REGION_DPAD_DL 32
+#define SE_REGION_DPAD_DOWN 33
+#define SE_REGION_DPAD_DR 34
+#define SE_REGION_MENU 35
+#define SE_REGION_MENU_HOVER 36
+#define SE_REGION_MENU_ACTIVE 37
+#define SE_REGION_MAX_REWIND 38
+#define SE_REGION_MAX_REWIND_HOVER 39
+#define SE_REGION_MAX_REWIND_ACTIVE 40
+#define SE_REGION_REWIND 41
+#define SE_REGION_REWIND_HOVER 42
+#define SE_REGION_REWIND_ACTIVE 43
+#define SE_REGION_PLAY 44
+#define SE_REGION_PLAY_HOVER 45
+#define SE_REGION_PLAY_ACTIVE 46
+#define SE_REGION_PAUSE 47
+#define SE_REGION_PAUSE_HOVER 48
+#define SE_REGION_PAUSE_ACTIVE 49
+#define SE_REGION_FF 50
+#define SE_REGION_FF_HOVER 51
+#define SE_REGION_FF_ACTIVE 52
+#define SE_REGION_MAX_FF 53
+#define SE_REGION_MAX_FF_HOVER 54
+#define SE_REGION_MAX_FF_ACTIVE 55
+#define SE_REGION_BLANK 56
+#define SE_REGION_BLANK_HOVER 57
+#define SE_REGION_BLANK_ACTIVE 58
+#define SE_REGION_VOL_EMPTY 59
+#define SE_REGION_VOL_EMPTY_ACTIVE 60
+#define SE_REGION_VOL_FULL 61
+#define SE_REGION_VOL_FULL_ACTIVE 62
+#define SE_REGION_VOL_KNOB 63
+#define SE_REGION_VOL_KNOB_ACTIVE 64
+#define SE_REGION_MENUBAR 65
+#define SE_TOTAL_REGIONS 66
+
+#define SE_RESIZE_STRETCH 0
 #define SE_RESIZE_FIXED 0x80
 
-#define SE_SCREEN_NONE  0
-#define SE_SCREEN_BOTH  0xC0
+#define SE_SCREEN_NONE 0
+#define SE_SCREEN_BOTH 0xC0
 
 #define SE_NO_SORT 0
 #define SE_SORT_ALPHA_ASC 1
 #define SE_SORT_ALPHA_DESC 2
 
-typedef struct{
+typedef struct {
   uint16_t start_pixel;
   uint16_t end_pixel;
   uint8_t resize_control;
   uint8_t screen_control;
   uint8_t gamepad_control;
-}se_control_point_t;
+} se_control_point_t;
 
-typedef struct{
-  int x,y;
-  int w,h;
+typedef struct {
+  int x, y;
+  int w, h;
   bool active;
   se_control_point_t control_points_x[SE_MAX_CONTROL_POINTS];
   se_control_point_t control_points_y[SE_MAX_CONTROL_POINTS];
-}se_theme_region_t;
+} se_theme_region_t;
 
-typedef struct{
+typedef struct {
   sg_image image;
   uint32_t im_w;
   uint32_t im_h;
-  uint32_t version_code; 
-  uint8_t palettes[5*4];
+  uint32_t version_code;
+  uint8_t palettes[5 * 4];
```

Comment:
* and / should not be wrapped in spaces (but + and - should be wrapped in spaces) to improve readability.

Reply:
uhm ok sounds like a good idea

Reply:
clang format does unfortunately not seem to support this (?)
is this a deal breaker? It seems to want to have spaces between most operators.